### DISCUSSION TITLE
add Coinex Smart Chain mainnet and testnet to wagmi

### DISCUSSION THREAD

**Discussion by thanhtx1178793** [wagmi-dev/references#90] [General]
Mainnet: 
- id: 52
- name: CoinEx Smart Chain Mainnet
- network: CoinEx Smart Chain Mainnet
- nativeCurrency: cet
- public RPC: https://rpc.coinex.net
- block explorer: https://www.coinex.net/

Testnet: 
- id: 53
- name:CoinEx Smart Chain Testnet
- network: CoinEx Smart Chain Testnet
- nativeCurrency: cett
- public RPC: https://testnet-rpc.coinex.net
- block explorer: https://testnet.coinex.net/

**Comment by tmm** [comment]
Please open a pull request adding the Chain.
